data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/GetSyncCommitteeContribution.java

```diff
@@ -0,0 +1,123 @@
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.SLOT;
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.SUBCOMMITTEE_INDEX;
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.TAG_EXPERIMENTAL;
+import static tech.pegasys.teku.beaconrestapi.SingleQueryParameterUtils.getParameterValueAsBytes32;
+import static tech.pegasys.teku.beaconrestapi.SingleQueryParameterUtils.getParameterValueAsInt;
+import static tech.pegasys.teku.beaconrestapi.SingleQueryParameterUtils.getParameterValueAsUInt64;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.javalin.http.Context;
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.tuweni.bytes.Bytes32;
+import org.jetbrains.annotations.NotNull;
+import tech.pegasys.teku.api.DataProvider;
+import tech.pegasys.teku.api.ValidatorDataProvider;
+import tech.pegasys.teku.api.response.v1.validator.GetSyncCommitteeContributionResponse;
+import tech.pegasys.teku.api.schema.altair.SyncCommitteeContribution;
+import tech.pegasys.teku.beaconrestapi.handlers.AbstractHandler;
+import tech.pegasys.teku.beaconrestapi.schema.BadRequest;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.provider.JsonProvider;
+
+public class GetSyncCommitteeContribution extends AbstractHandler {
+  public static final String ROUTE = "/eth/v1/validator/sync_committee_contribution";
+
+  private final ValidatorDataProvider provider;
+
+  public GetSyncCommitteeContribution(
+      final DataProvider dataProvider, final JsonProvider jsonProvider) {
+    super(jsonProvider);
+    this.provider = dataProvider.getValidatorDataProvider();
+  }
+
+  @OpenApi(
+      path = ROUTE,
+      method = HttpMethod.GET,
+      summary = "Produce sync committee contribution",
+      tags = {TAG_EXPERIMENTAL},
+      queryParams = {
+        @OpenApiParam(
+            name = SLOT,
+            description =
+                "`uint64` The slot for which a sync committee contribution should be created.",
+            required = true),
+        @OpenApiParam(
+            name = SUBCOMMITTEE_INDEX,
+            description = "`uint64` The subcommittee index for which to produce the contribution.",
+            required = true),
+        @OpenApiParam(
+            name = BEACON_BLOCK_ROOT,
+            description = "`bytes32` The block root for which to produce the contribution.",
+            required = true)
+      },
+      description =
+          "Returns an unsigned sync committee contribution, for an validator to sign and post.",
```

Comment:
```suggestion
          "Returns an unsigned sync committee contribution, for a validator to sign and post.",
```

Although I think I probably would have said:
```
Returns a `SyncCommitteeContribution` that is the aggregate of `SyncCommitteeSignature` values known to this node matching the specified slot, subcommittee index and beacon block root.
```